keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -2320,6 +2192,18 @@ successfully before allowing the pod to be deleted. This ensures that network-at
 for other Pods, including those that might get scheduled to other nodes. It
 also signals that it is safe to deallocate and delete the ResourceClaim.
 
+This implies that a DRA driver must not get removed while it is in use on a
+node because otherwise pods get stuck while the kubelet (re)tries to call
+NodeUnprepareResources. A driver is in use if it has ResourceClaims which
+were prepared. The kubelet exposes that in a metric:
+- Metric name: `dra_resource_claims_in_use`
+- Description: The number of ResourceClaims that are currently in use on the node, by driver name (`driver_name` label value) and across all drivers (special value `<any>` for `driver_name`). Note that the sum of all by-driver counts is not the total number of in-use ResourceClaims because the same ResourceClaim might use devices from different drivers. Instead, use the count for the `<any>` driver_name.
+- Type: Histogram
+- Labels: `driver_name`
```

Comment:
The implementation is ready, see https://github.com/kubernetes/kubernetes/pull/131641. I'm holding it for review of the KEP update.

The design and implementation of this metrics was reviewed by @richabanker.

It starts out as alpha, which is consistent with the other metrics (see above).